### PR TITLE
PSEA Update

### PR DESCRIPTION
**Pull request by reinbach**
Update assessments to make use of TRANSITION_SIDE_EFFECTS

### CODE REVIEW

**Comment by robertavram** [comment]
does it work as expected? @reinbach ? :)

**Comment by reinbach** [comment]
@robertavram yes it is, there are tests in place, which continued to pass after changing to this method/logic

**Comment by robertavram** [comment]
super.. I've made some changes to the old version.. once merged in develop maybe we can adjust this one :D ... it seems like nested objects dont ref well.

File: src/etools/applications/psea/validation.py [hunk 1]
```diff
@@ -102,3 +105,35 @@ def assessment_focal_point_user(assessment, user):
...
+        recipients=assessment.get_focal_recipients(),
+        template_name="psea/assessment/submitted",
+        context=assessment.get_mail_context(user)
+    )
+
+
+def assessment_rejected(assessment, old_instance=None, user=None):
+    send_notification_with_template(
+        recipients=assessment.get_recipients(),
+        template_name="psea/assessment/rejected",
```
Comment (robertavram) [inline src/etools/applications/psea/validation.py]: @reinbach I think this template is also looking for "rejection_comment" and will end up blank

File: src/etools/applications/psea/views.py [hunk 2]
```diff
@@ -230,44 +228,7 @@ def _set_status(self, request, assessment_status):
...
-                )
-            # assigned status is automatically moved to in progress
-            elif instance.status == Assessment.STATUS_IN_PROGRESS:
-                send_notification_with_template(
-                    recipients=instance.assessor.get_recipients(),
-                    template_name="psea/assessment/assigned",
-                    context=context
-                )
-            elif instance.status == Assessment.STATUS_REJECTED:
-                context["rejected_comment"] = instance.get_rejected_comment()
```
Comment (robertavram) [inline src/etools/applications/psea/views.py]: this one :)

File: src/etools/applications/psea/validation.py (hunk 1 above)
Comment (reinbach) [reply src/etools/applications/psea/validation.py]: Oops :(

**Pull request merged**